spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/ConsumerProperties.java

```diff
@@ -125,5 +127,13 @@ public HeaderMode getHeaderMode() {
 	public void setHeaderMode(HeaderMode headerMode) {
 		this.headerMode = headerMode;
 	}
+
+	public boolean isSupportDeserializationNatively() {
```

Comment:
Given the native deserializer would have already kicked in before the ReceivingHandler at the consumer binding receives the message, there is no way of bypassing the codec at the consumer side I think.